databases/postgresql-cloudnativepg/manifests/01-basic-cluster/postgreSQL_cluster.yaml

```diff
@@ -0,0 +1,72 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START gke_manifests_postgresql_cluster]
```

Comment:
Could you use this prefix for all the region tags?
`gke_databases_postgresql_cloudnativepg_FILE_NAME`

And make sure there's no empty line after a `START` or before an `END`

Reply:
@bourgeoisor does it apply to terraform files too?

Reply:
Ideally the prefix is more-or-less the directory path to the file

Reply:
These should all have the same prefix since they're in the same samples directory

![image](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/assets/3271352/dad91f61-bd67-4049-b244-5ae3029f4356)

Reply:
Oh I see - gke_databases_postgresql_cloudnativepg_**FILE_NAME** this FILE_NAME confused me

Reply:
Wait now it's worse 😁 

Let me rephrase: If there's only one region tag in a file, then the region tag should be more-or-less "gke" + the path + the filename. It's fine to shorten some things though (for example shortening "01-basic-cluster" to just "01".

For example...

If the file is `databases/postgresql-cloudnativepg/manifests/01-basic-cluster/postgreSQL_cluster.yaml`

Then the tag should be something along the lines of `gke_databases_postgresql_cloudnativepg_manifests_01_postgresql_cluster`

Each tag should be unique. Does that make more sense?

Reply:
Definitely, give me a moment to fix that

Reply:
and what about files with two region tags?

Reply:
now it should be fine

Reply:
If there's two region tags in the same file then you can add another suffix to differentiate them, for example if it's a file with a Role and then a RoleBinding you could have:

- gke_path_to_file_role
- gke_path_to_file_rolebinding

Reply:
Looks good so far, just one last fix, it looks like there's 2 tags that are the same in 2 files (so 4 tags total):

![image](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/assets/3271352/ee9d596f-1607-4b35-a53c-68657423d6e8)

What happens (and the reason tags need to be unique) is that if we embed `gke_databases_postgresql_cloudnativepg_autopilot_private_regional_cluster` inside the docs / tutorial, it won't know which of the two files to show the users and might pick the wrong one

Reply:
ok got it, uniqueness applied

Reply:
This START doesn't match its END:

https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/e78f2ca3ee07ad44714be34a887bd3b50a86ea48/databases/postgresql-cloudnativepg/terraform/modules/network/main.tf#L15

https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/e78f2ca3ee07ad44714be34a887bd3b50a86ea48/databases/postgresql-cloudnativepg/terraform/modules/network/main.tf#L70

Reply:
done